mcs/class/corlib/System.Reflection/Module.cs

```diff
@@ -446,5 +436,9 @@ public virtual Type[] GetTypes()
 		public virtual IEnumerable<CustomAttributeData> CustomAttributes {
 			get { return GetCustomAttributesData (); }
 		}
+
+		internal virtual IntPtr GetImpl () {
```

Comment:
Can't we just mark this method as `abstract` since the class is already `abstract`?

Reply:
Fixed.